content/en/docs/reference/config/security/authorization-policy/index.html

```diff
@@ -143,7 +143,7 @@
 <p>{{<tabset category-name="example">}}
 {{<tab name="v1beta1" category-value="v1beta1">}}</p>
 <pre><code class="language-yaml">apiVersion: security.istio.io/v1beta1
-kind: AuthorizaionPolicy
+kind: AuthorizationPolicy
```

Comment:
I think this is not the right place to fix the typo, these are just copied yamls from `istio/api` and should be fixed there.

Reply:
See https://github.com/istio/api/blob/master/security/v1/authorization_policy.proto#L177

Reply:
Nice point, ty